webpack.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         rules: [
             {test: /\.scss$/, use: [MiniCssExtractPlugin.loader, "css-loader", "sass-loader"]},
             {test: /\.js|.jsx$/, exclude: /node_modules/, use: "babel-loader"},
-            {test: /\.(png|jpe?g|gif|svg)$/i, use: [{loader: "file-loader"}]}
+            {test: /\.(png|jpe?g|gif|svg)$/i, use: [{loader: "file-loader"}]},
```

Comment:
Did this sneak in because you made some other modification to the webpack.config that you subsequently undid? If, as a guess, you were attempting to include the bootstrap .css and were running into webpack configuration problems, you could solve that by changing the test string matcher in line 23 above to `/\.(scss|css)$/`.

Reply:
I'm not sure how that comma got there, but we were running into webpack issues with `bootstrap.css`. Things seemed to work when we linked an absolute URL to the CSS file in the `index.html` template in the backend directory, else we would only see the browser's default styling for the page. This still occurs even with the change to line 23, which I committed along with deleting the extra comma.

Reply:
That's strange. Where are you attempting to import the bootstrap css, and what's the error you're receiving? It makes sense that directly importing the css from the CDN would solve the immediate error, since that's bypassing webpack entirely, but it'd be a good idea to figure out why we can't import the file as expected given our configuration.

Reply:
We're not necessarily getting an error, but none of the bootstrap styling takes effect. We just get plain HTML pages with the browser's default styling. We can try experimenting further to see if we get real errors!